src/boot.lisp

```diff
@@ -281,6 +281,25 @@
                                   (list (first v) (third v))))
                            varlist)))))))
 
+(defmacro declare (&rest declarations)
```

Comment:
If I recall correctly, I added some `(declare (ignore …))` to silence SBCL warnings in the host pass, which threw out some error about undefined function `declare` during the target pass.